docker-compose.yml.tmpl

```diff
@@ -24,9 +19,6 @@ services:
 
   anthias-server:
     image: screenly/anthias-server:${DOCKER_TAG}-${DEVICE_TYPE}
-    build:
-      context: .
-      dockerfile: docker/Dockerfile.server
```

Comment:
I'm not sure if I will remove this one. Running `docker compose` with the compose file generated from this template will fail. At the moment, I'm just running `bin/build_containers.sh` when re-building the images/services.